src/pages/billing/Project.js

```diff
@@ -444,17 +493,25 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
           _.slice(0, maxWorkspacesInChart)
         )(workspaceDetails?.spendData)
         // Pull out names and costs.
-        const costsPerWorkspace = { workspaceNames: [], workspaceCosts: [], numWorkspaces: workspaceDetails?.spendData.length }
+        const costPerWorkspace = {
+          workspaceNames: [], workspaceCosts: [], computeCosts: [], storageCosts: [], otherCosts: [],
+          costFormatter, numWorkspaces: workspaceDetails?.spendData.length
+        }
         _.forEach(workspaceCostData => {
-          costsPerWorkspace.workspaceNames.push(workspaceCostData.workspace.name)
-          costsPerWorkspace.workspaceCosts.push(parseFloat(workspaceCostData.cost))
+          costPerWorkspace.workspaceNames.push(workspaceCostData.workspace.name)
```

Comment:
To avoid exposing rounding errors, we may wish to obtain the total workspace cost from compute + storage + other. See https://broadinstitute.slack.com/archives/C02L9L96YB1/p1647869664432039?thread_ts=1647818613.734029&cid=C02L9L96YB1.